CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 Not yet released; provisionally v2.0.0 (may change).
 
+
+### GetLeavesByRevisionNoProof API
+Allow map clients to forgo fetching inclusion proofs.
+This dramatically speeds things up for clients that don't need verifiability.
```

Comment:
Not sure if it's worth giving an example here.

Reply:
I don't think it is?

Reply:
I just meant a sentence on why you might want to do this but it's not really important.